### PR TITLE
overlay header as well for better focus

### PR DESCRIPTION
**Pull request by jancborchardt**
Please review @tomneedham @owncloud/designers 

In addition it would be cool to also implement »Close when clicking outside of editor« #132

### CODE REVIEW

**Comment by tomneedham** [comment]
Just checking this out, not sure what you've changed?!

File: css/style.css [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	position: absolute;
 	top: 0;
 	width: 100%;
-	z-index: 1000;
+	z-index: 3000;
```
Comment (MorrisJobke) [inline css/style.css]: @tomneedham 1 -> 3

File: css/style.css [hunk 2]
```diff
@@ -40,20 +40,20 @@
 }
 
 #editor_container{
-    position: fixed;
-    top: 8%;
+	position: fixed;
+	top: 6%;
```
Comment (MorrisJobke) [inline css/style.css]: @tomneedham 8 -> 6

File: css/style.css [hunk 3]
```diff
@@ -40,20 +40,20 @@
...
+	z-index: 9999;
+	overflow: hidden;
+	background-color: #fff;
 	border-radius: 3px;
 	box-sizing: border-box;
 }
 
 #editor_container.hasPreview {
-	z-index: 1002;
+	z-index: 3002;
```
Comment (MorrisJobke) [inline css/style.css]: @tomneedham 1 -> 3

File: css/style.css [hunk 4]
```diff
@@ -173,12 +173,12 @@
...
     position: fixed;
-    top: 0%;
-    left: 0%;
+    top: 0;
+    left: 0;
     width: 100%;
     height: 100%;
     background-color: black;
-    z-index:1001;
+    z-index: 3001;
```
Comment (MorrisJobke) [inline css/style.css]: @tomneedham 1 -> 3

**Comment by MorrisJobke** [comment]
@tomneedham I highlighted the changes for you ;)

File: css/style.css (hunk 1 above)
Comment (Henni) [inline css/style.css]: @jancborchardt could you please add a css comment why the z-index has to be exactly 3000.

**Comment by jancborchardt** [comment]
@Henni added the comment about the z-index.

@tomneedham you can see the diff without whitespace here: https://github.com/owncloud/files_texteditor/pull/138/files?w=0

**Comment by tomneedham** [comment]
but what's the implication for the user? Everything appears the same when using the editor.

**Comment by jancborchardt** [comment]
Currently:
capture du 2015-11-11 17-22-46
This change:
capture du 2015-11-11 17-21-54

It provides better focus on the text file by also overlaying the header. Same as we do with the video player by @Instagit https://github.com/Instagit/videojs-sublime/pull/15

**Comment by tomneedham** [comment]
:+1:

**Pull request merged**